modules/python/examples/ukf-nonlinear-complex-example.py

```diff
@@ -118,48 +118,44 @@ def measurementMean(measurements: List[ColVector], wm: List[float]) -> ColVector
       sumCos[j] += np.cos(measurements[i][(2*j)+1]) * wm[i]
       sumSin[j] += np.sin(measurements[i][(2*j)+1]) * wm[i]
 
-  for j in range(nbLandmarks):
-    mean[(2*j)+1] = np.arctan2(sumSin[j], sumCos[j])
+  orientations = np.arctan2(sumSin, sumCos)
+  mean[1::2] = orientations[0::1]
```

Comment:
you can omit the [0::1]: this is the default behavior (start from, iterate until the end with a step of one)